packages/components/src/components/dropdown/dropdown.jsx

```diff
@@ -122,10 +123,15 @@ class Dropdown extends React.PureComponent {
         if (item.value !== this.props.value) {
             this.props.onChange(
                 { target: { name: this.props.name, value: item.value } });
+            this.updateSelected(item.value);
         }
         this.handleVisibility();
     };
 
+    updateSelected(value) {
```

Comment:
@negar-binary this can be removed.

Reply:
I don't agree. it's better not to duplicate the code and have one function that does it.

Reply:
It’s one line of code which set a value. It’s like you create a function to set a=b. But not a big deal if you insist to keep it.